### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by JoseI11**
# Summary

- I made the component navbar to navigate with its responsive aspect 
- I check the navbar with a cypress test
...

# Detail

If needed, a detailed explanation of what the pr is about.

# Evidence

Vite-React-TS-—-Mozilla-Firefox-2023-12-13-22-42-39

# Reference.

tarea jira navbar

### CODE REVIEW

File: cypress/screenshots/components/navbar.cy.ts/Navbar test -- Should show the navbar (failed).png
Comment (Fludeo) [inline cypress/screenshots/components/navbar.cy.ts/Navbar test -- Should show the navbar (failed).png:1]: We should not send all these pngs. Pls add to the .gitignore file the screenshot folder

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 import Login from './pages/Login';
-import HomeTest from './pages/HomeTest';
 
 const App: React.FC = () => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<HomeTest />} />{' '}
+        <Route path="/Home" element={<Home />} />{' '}
```
Comment (Fludeo) [inline src/App.tsx]: pls change "Home" to "home" and keep the lowercase at the beginning of every route. Use kebab case for multiple words.

**Review by Fludeo — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (JoseI11) [reply src/App.tsx]: Done

File: cypress/screenshots/components/navbar.cy.ts/Navbar test -- Should show the navbar (failed).png
Comment (JoseI11) [reply cypress/screenshots/components/navbar.cy.ts/Navbar test -- Should show the navbar (failed).png:1]: Done, I add the folder screenshots to gitignore

**Review by Fludeo — APPROVED** [review]
Great.

**Pull request merged**